src/schema/v2/sale_artworks.ts

```diff
@@ -115,7 +115,7 @@ export const SaleArtworksConnectionField: GraphQLFieldConfig<
     _root,
     {
       // artistIDs,
-      // includeArtworksByFollowedArtists,
+      includeArtworksByFollowedArtists = true,
```

Comment:
We should use the arguments definition to expose default variables in the schema, for example:

```tsx
includeArtworksByFollowedArtists: {
  type: GraphQLBoolean,
  defaultValue: true
},
```

But I don't think that `true` is a good default here, due to this TODO comment above in the file:

https://github.com/artsy/metaphysics/blob/e1c227edc16ac0f38ab28db73c32c26bd8b9d5d7/src/schema/v2/sale_artworks.ts#L74-L77

I've [asked on Slack](https://artsy.slack.com/archives/CP9P4KR35/p1599681100120300) for suggestions here, since I'm not sure what exactly the best solution is.